From 411a0e512344fd4f4da29cb79dbe7bffecacf93e Mon Sep 17 00:00:00 2001
From: Giuseppe Maggio <jertlok@proton.me>
Date: Sun, 9 Aug 2026 17:24:24 +0200
Subject: [PATCH 088/113] dt-bindings: sound: qcom,sm8250: add the msm8998
 sound card

The msm8998 machine driver binds on "qcom,msm8998-sndcard" and shares
the properties this schema already describes for the other q6dsp-based
cards.

Signed-off-by: Giuseppe Maggio <jertlok@proton.me>
---
 Documentation/devicetree/bindings/sound/qcom,sm8250.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml b/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
index b49a920af704..f57e24573fb5 100644
--- a/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
+++ b/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
@@ -32,6 +32,7 @@ properties:
           - fairphone,fp5-sndcard
           - qcom,apq8096-sndcard
           - qcom,glymur-sndcard
+          - qcom,msm8998-sndcard
           - qcom,qcm6490-idp-sndcard
           - qcom,qcs615-sndcard
           - qcom,qcs6490-rb3gen2-sndcard
